R/missing_argument_linter.R

```diff
@@ -31,7 +31,7 @@ missing_argument_linter <- function(except = c("switch", "alist")) {
           column_number = col1[[i]],
           type = "warning",
           message = "Missing argument in function call.",
-          line = source_file$lines[line1[[i]]],
```

Comment:
Thanks for explaining.
I'd call this print unexpected, since we want the line to be readable:

```
<path>:2:3: warning: Missing argument in function call.
NA
  ^
```

Reply:
Agree.